crates/rust-analyzer/src/config.rs

```diff
@@ -307,6 +307,8 @@ config_data! {
         /// Join lines unwraps trivial blocks.
         joinLines_unwrapTrivialBlock: bool = "true",
 
+        /// Where to render annotations.
+        lens_annotationLocation: AnnotationLocation = "\"above_name\"",
```

Comment:
```suggestion
        lens_location: AnnotationLocation = "\"above_name\"",
```
lens and annotation are the same thing for us